jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics.internal.container;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.metrics.internal.Constants;
+import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+public final class ContainerConfigurationHandler implements RecordedEventHandler {
+  public static final String EVENT_NAME = "jdk.ContainerConfiguration";
```

Comment:
Please go through all the static final fields in the PR to make private the ones that aren't used outside the class